src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
```

Comment:
To be flexible, we should not add the assumption the lab script location is the same as this script.

@roncain, My goal is to keep the script simple and small so it's easier to read and modify. This script is only used for Helix lab automation by a schedule task, not used by Dev or CI experience. That's why I intentionally leave checking parameters out. I'm open to add it though if you feel strongly we should do it.